### PR TITLE
Fixed null.client runtime in vote.dm

### PR DESCRIPTION
**Pull request by Jujumatic**
Turns out doing votes in the lobby screen can cause null.client errors - I have added a check to fix this

:cl:
fix: Fixed a null client error if a vote is started in the lobby screen
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/vote.dm [hunk 1]
```diff
@@ -337,4 +338,5 @@ SUBSYSTEM_DEF(vote)
 		if(P)
 			P.player_actions -= src
 	else
-		return
\ No newline at end of file
+		return
+*/	Hippie end
```
Comment (Carbonhell) [inline code/controllers/subsystem/vote.dm]: dude that's not how comments work, */ is the END of the comment

**Review by Carbonhell — CHANGES_REQUESTED** [review]

File: hippiestation/code/controllers/subsystem/vote.dm [hunk 2]
```diff
@@ -234,4 +234,17 @@
...
+		if(owner.client)
+			owner.client.player_details.player_actions -= src
+		else if(owner.ckey)
+			var/datum/player_details/P = GLOB.player_details[owner.ckey]
+			if(P)
+				P.player_actions -= src
+		else
+			return
+	else
+		return
```
Comment (Carbonhell) [inline hippiestation/code/controllers/subsystem/vote.dm]: else return is pointless

Comment (Jujumatic) [reply hippiestation/code/controllers/subsystem/vote.dm]: if they're not owner then return

that's not needed?

Comment (Carbonhell) [reply hippiestation/code/controllers/subsystem/vote.dm]: ain't returning anything, and there is no code after it,so what is the point of this and the return before?

**Review by Carbonhell — APPROVED** [review]

**Pull request merged**